crates/frontend/src/components/context_card.rs

```diff
@@ -0,0 +1,80 @@
+use leptos::*;
+use serde_json::{Map, Value};
+
+use crate::{
+    components::{
+        condition_pills::{types::Condition, ConditionPills},
+        table::{types::Column, Table},
+    },
+    types::Context,
+};
+
+#[component]
+pub fn context_card(
+    context: Context,
+    overrides: Vec<(String, Value)>,
+    handle_edit: Callback<(Context, Vec<(String, Value)>), ()>,
```

Comment:
Maybe we can type alias overrides instead of writing Map<String,Value> everywhere